modules/rosa-sts-creating-cluster.adoc

```diff
@@ -135,17 +136,21 @@ I: To watch your cluster installation logs, run 'rosa logs install -c <cluster_n
 [source,terminal]
 ----
 $ rosa create cluster \
-  --cluster-name ${name} \
-  --region ${region} \
-  --version ${version} \
-  --role-arn arn:aws:iam::${aws_account_id}:role/ManagedOpenShift-Installer-Role \ <1>
-  --support-role-arn arn:aws:iam::${aws_account_id}:role/ManagedOpenShift-Support-Role \
-  --master-iam-role arn:aws:iam::${aws_account_id}:role/ManagedOpenShift-ControlPlane-Role \
-  --worker-iam-role arn:aws:iam::${aws_account_id}:role/ManagedOpenShift-Worker-Role \
-  --operator-roles-prefix ManagedOpenShift <2>
+  --cluster-name <cluster_name> \ <1>
+  --region <aws_region> \ <2>
+  --version <openshift_version> \ <3>
+  --role-arn arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-Installer-Role \ <4><5>
+  --support-role-arn arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-Support-Role \
+  --master-iam-role arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-ControlPlane-Role \
+  --worker-iam-role arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-Worker-Role \
+  --operator-roles-prefix ManagedOpenShift <6>
 ----
-<1> If you specified a custom prefix in the preceding command, you must replace the `ManagedOpenShift` prefix with the custom one in each ARN declaration. You must specify ARNs for STS account-wide roles that are created using `rosa create account-roles`.
-<2> Declares the prefix for the cluster-specific Operator roles that are defined in the following step.
+<1> Replace `<cluster_name>` with the name of your cluster.
+<2> Replace `<aws_region>` with the name of the AWS region, for example `us-east-1`.
+<3> Replace `<openshift_version>` with the OpenShift version, for example `4.8.9`.
```

Comment:
We usually don't use bare "OpenShift," so please double-check that this is more appropriate than "{product-name}."

Reply:
@kalexand-rh thank you for your review!

In the OSD docs, `{product-title}` resolves as "Red Hat OpenShift Service on AWS CLI (rosa)". The `--version` option in the `rosa create cluster` command relates specifically to the OpenShift version, not the ROSA version. This is outlined in the help menu for that sub-command:

```
$ rosa create cluster --help | grep -is 'version'
      --version string                 Version of OpenShift that will be used to install the cluster, for example "4.3.10"
```

Reply:
That's good to know! Thank you.